tests/dedupe_test.py

```diff
@@ -515,8 +515,8 @@ def test_import_service(self):
         driver.find_element(By.PARTIAL_LINK_TEXT, "Path Test 1").click()
         driver.find_element(By.ID, "dropdownMenu1").click()
         driver.find_element(By.LINK_TEXT, "Re-Upload Scan").click()
-        driver.find_element(By.XPATH, '//*[@id="base-content"]/form/div[3]/div/div').click()
-        driver.find_element(By.XPATH, '//*[@id="base-content"]/form/div[4]/div/div').click()
+        # driver.find_element(By.XPATH, '//*[@id="base-content"]/form/div[3]/div/div').click()
+        # driver.find_element(By.XPATH, '//*[@id="base-content"]/form/div[4]/div/div').click()
```

Comment:
please remove them or add a comment that explicitly say that we don't want to force active/verified